docs/sql-ref-syntax-qry-select-groupby.md

```diff
@@ -91,6 +93,18 @@ aggregate_name ( [ DISTINCT ] expression [ , ... ] ) [ FILTER ( WHERE boolean_ex
     Filters the input rows for which the `boolean_expression` in the `WHERE` clause evaluates
     to true are passed to the aggregate function; other rows are discarded.
 
+* **FIRST**
+
+    `FIRST` select a first expression value from the data set, we can specific `IGNORE NULLS` to ignore NULLS, it is optional. 
```

Comment:
I guess something like this 
```
`FIRST` selects a first expression value from the data set. We can specify an optional `IGNORE NULL` clause to ignore NULL values.
``` 
Probably also change the following sentences?

Reply:
it seems your expression more natural. will improve it.